rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @rentals = rentals
+    @book = book
+    book.rentals = self
+    person.rentals = self
```

Comment:
:red_circle:  [REQUIRED] I just wanted to let you know that in Ruby, you use the `<<` sign to add new items to an array, not the `=` sign. For example, the following code would add the string "hello" to the end of the array `array`:
```

array = ["world"]
array << "hello"
```
The `<<` sign is a shortcut for the push method, which is used to add new items to an array. You can also use the push method, but the `<<` sign is a bit more concise.